### PR TITLE
Support for rpm packages and various Linux fixes

### PR DESCRIPTION
**Pull request by fcastilloec**
I've added support for `rpm` packages. I've tested them on Fedora/26 and Fedora/25 and there are no problems in either of those systems.
The Travis script was updated to deploy for a repository on Fedora/26.
To be able to test the `rpm` (Fedora) repository, we'll have to merge this PR, create a branch and publish that branch (by changing `branch` tag on` `.travis.yml`) and if this is working correctly, that branch can be deleted. If there are errors, we can fix in that branch and then merge.

Last, I updated the README to better describe how to build from source.

### CODE REVIEW

File: debian/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "electron": "^1.6.11",
     "electron-installer-debian": "^0.5.0",
+    "electron-installer-redhat": "^0.5.0",
     "electron-packager": "^7.0.2",
     "electron-rebuild": "^1.6.0",
     "foreman": "^2.0.0"
```
Comment (danielravina) [inline debian/package.json]: should we just rename `debian/` directory to `linux/`?

File: .travis.yml [hunk 2]
```diff
@@ -22,13 +23,25 @@ cache:
...
+    repository: "headset-electron"
+    username: "headsetapp"
+    token: "${PACKAGECLOUD_TOKEN}"
+    dist: "fedora/26"
+    local_dir: build/installers
+    package_glob: "*.rpm"
+    skip_cleanup: true
+    on:
+      branch: master
+      tags: true
```
Comment (danielravina) [inline .travis.yml]: noice!

File: debian/package.json (hunk 1 above)
Comment (fcastilloec) [inline debian/package.json]: That's a good idea, I can change that.

File: .travis.yml (hunk 2 above)
Comment (fcastilloec) [inline .travis.yml]: Once it's tested that the repo is working, I can get instructions on how to add it manually. I'm not as familiar with `rpm` repositories but I don't think it's that hard.

**Comment by fcastilloec** [comment]
Once this is merged, and a new version released. A couple of issues can be closed, specifically: #5 #15 and #55

**Comment by danielravina** [comment]
🎉🎉🎉 Great thanks a lot @fcastilloec!

**Comment by danielravina** [comment]
woops, just realized there are still references to packagecloud in the readme, I guess we could remove it in the next PR?

**Comment by fcastilloec** [comment]
Yes, when we setup the official repo we can change the README.

**Comment by joseluisq** [comment]
Cool!
Don't forget to update the website too! :+1:

**Pull request merged**